Uchu.Core/Enums/GameMasterLevel.cs

```diff
@@ -2,10 +2,11 @@ namespace Uchu.Core
 {
     public enum GameMasterLevel
     {
-        Player,
-        Mythran,
-        Admin,
-        Operator,
+        Player = 0,
+        Mythran = 1,
+        MytranWithGlow = 2,
```

Comment:
Typo in MytranWithGlow, should be MythranWithGlow

Reply:
Fixed